src/components/CheckUsage.tsx

```diff
@@ -31,7 +31,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
   }),
 });
 
-const hideTelemetryForTypes = [CheckType.Scripted, CheckType.MULTI_HTTP];
+const hideTelemetryForTypes = [CheckType.Scripted, CheckType.MULTI_HTTP, CheckType.Browser];
```

Comment:
I thought I'd add this whilst I was there (I was considering adding the alert about them being free to this component but decided against it in the end).